scripts/build.sh

```diff
@@ -127,32 +127,7 @@ mkdir -p $binary_cache
 # Create the minimal tar bundle
 tar czf $tmp_output_file -C $release_dir contiv-$VERSION
 
```

Comment:
Can we comment out the portion below rather than removing it? We might need it in future.